force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +68,17 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
-        
+        List <Boolean> isSFAwesome = new List <Boolean>();
         // Check if the string starts with 'Sales'
-        
+        Boolean startsWithSales = input.startsWith('Sales');
         // Check if the string ends with 'Force'
-        
+        Boolean endsWithForce = input.endsWith('Force');
         // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        Boolean containsAwesome = input.contains('Awesome');
+        isSFAwesome.add(startsWithSales);
+        isSFAwesome.add(endsWithForce);
+        isSFAwesome.add(containsAwesome);
+        return isSFAwesome; 
```

Comment:
Good work. The only thing I like to do differently is add the results of my String methods directly to my list. 

```
 public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
 ```